### PR TITLE
flowschema for oauth server and its operator

### PR DESCRIPTION
**Pull request by tkashem**
- `SAR` or `tokenreviews` from oauth 
- Any other requests from oauth server.
- Traffic from oauth operator.

See https://github.com/openshift/cluster-kube-apiserver-operator/pull/966 for more details.

### CODE REVIEW

File: manifests/09_flowschema.yaml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  distinguisherMethod:
+    type: ByUser
+  matchingPrecedence: 2000
+  priorityLevelConfiguration:
+    name: openshift-control-plane-operators
+  rules:
+  - resourceRules:
+    - apiGroups:
+      - '*'
+      clusterScope: true
```
Comment (deads2k) [inline manifests/09_flowschema.yaml:78]: will this include all namespaced requrests too?

Comment (tkashem) [reply manifests/09_flowschema.yaml:78]: Yes, that's how it was designed I think.

**Pull request merged**